go/service/gregor.go

```diff
@@ -1067,7 +1067,7 @@ func (g *gregorHandler) newChatActivity(ctx context.Context, m gregor.OutOfBandM
 
 		// We need to get this conversation and then localize it
 		var inbox chat1.Inbox
-		if inbox, _, err = g.G().InboxSource.ReadRemote(ctx, uid, nil, &chat1.GetInboxLocalQuery{
+		if inbox, _, err = g.G().InboxSource.ReadNoCache(ctx, uid, nil, &chat1.GetInboxLocalQuery{
```

Comment:
Technically this and the other callsite of `ReadNoCache` could use the new `ReadRemote`, but we can just leave as is for this PR. Maybe make a ticket?